src/main/java/com/soboleoko/medicalclinic/model/Patient.java

```diff
@@ -0,0 +1,119 @@
+
+package com.soboleoko.medicalclinic.model;
+
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import java.time.LocalDate;
+import java.util.Objects;
+
+
+@RequiredArgsConstructor
+@Data
+
+//brak getterów, setterów oraz konstruktora, String zamiast long, modyfikatory dostępu
+//jeżeli chcemy otrzymać indywidualne wartości to nie używamy beana
+public class Patient {
+
+    private String email;
+    private String password;
+    private String idCardNo;
+    private String firstName;
+    private String lastName;
+    private String phoneNumber;
+    private LocalDate birthday;
+
+    public Patient(String email, String password, String idCardNo, String firstName, String lastName, String phoneNumber, LocalDate birthday) {
```

Comment:
all of the code is not necessary, @Data and @AllArgsConstructor will work

Reply:
Nie piszemy boilerplate codeu generujmy go z uzyciem Lomboka